src/pages/using-spark/components/icon.mdx

```diff
@@ -39,6 +39,56 @@ give feedback.
 
 ## Accessibility
 
-Icons are a great way to provide additional meaning for clients
-with cognitive and visual disabilities. They should be
-accompanied by text whenever possible.
+The following guidance is based on the W3C guidelines and universal 
+accessibility best practices:  
+
+- Ensure all informative `<img>` elements have a meaningful alternate (ALT) 
+text and all decorative `<img>` elements have empty alt attributes `(e.g. alt="")`. 
+- There are three main ways to add alternate text to an image: 
+  - Using an alt attribute. Example: `<img alt="drawing of a cat" src="...">`
+  - Using an aria-label. Example: `<img aria-label="drawing of a cat" src="...">`
+  - Using an aria-labelledby attribute. Example: `<img arialabelledby="someID" src="...">`
+
+- Alt text should give the intent, purpose, and meaning of the image. 
+
+- When writing the alternative text for an informative image always 
+consider how the user will experience it based upon context of the entire page. 
+
+- When writing the alternative text for a functional/active image 
+(such as clickable icons or buttons), make sure that it helps your users understand what happens when they click the image. 
+
+- Length 
+  - 125 characters or less. 
+  - Be as concise as possible while providing meaningful descriptions. 
+  - Don’t include phrases like “image of” or “picture of.” 
+  - Don’t include copyright or source information. 
+  - Alternative text will innately improve the discoverability of your page. 
+However, always remember that the purpose of ALT text is to provide accessibility 
+for disabled users. (For example, it will empower your blind users to experience 
+the image through the use of assistive technology, such as screen readers). 
+Hence, you should never use ALT text as a SEO opportunity. 
+You can include keywords only if they help describe the image. 
+
+- Style
+  - Use sentence case. 
+  - Include a period at the end of the alt text – even if it’s not a 
```

Comment:
Hey @jugrinovskiy ! Woot! First PR! I noticed here was to include a period at the end even its not a complete sentence but in the example up above, we have "drawing of a cat" and no period. Do we want to update our example to have one?

Reply:
**** I agree. Also, line 63 says not to include phrases like "image of" and our examples use "drawing of".